pkg/middleware/auth_proxy.go

```diff
@@ -31,20 +31,23 @@ func initContextWithAuthProxy(store *remotecache.RemoteCache, ctx *m.ReqContext,
 
 	// Check if allowed to continue with this IP
 	if result, err := auth.IsAllowedIP(); !result {
+		ctx.Logger.Error("auth proxy: failed to check whitelisted ip addresses", "error", err)
```

Comment:
Since `err` in this case is a typed error I think we should log `err.DetailsError` as well.

Reply:
Fixed